Library/src/Multiplayer/EventSerialisation.cpp

```diff
@@ -60,12 +64,19 @@ std::string RemoveIdPrefix(const std::string& Id)
 	return Id;
 }
 
+csp::common::String DecodeSequenceKey(csp::multiplayer::ReplicatedValue& RawValue)
+{
+	// Sequence keys are URI encoded to support reserved characters.
+	return csp::common::Decode::URI(RawValue.GetString());
+}
+
 } // namespace
 
 csp::common::String csp::multiplayer::GetSequenceKeyIndex(const csp::common::String& SequenceKey, int i)
 {
 	const std::string SequenceKeyString(SequenceKey.c_str());
-	// Match item after second ':' to get our parent id
+	// Match item after second ':' to get our parent Id.
```

Comment:
"i" should probably be unsigned

Reply:
I agree with @MAG-ME in this case. Making it unsigned will provide additional protection in the form of warnings at compile time should a developer attempt to pass a signed integer in. Will make the change.

Reply:
Fixed in latest.